examples/staking-broker/src/wasm.rs

```diff
@@ -0,0 +1,187 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+//
+// This is a very trivial implementation of a staking broker that would accept
+// users messages and bond, unbond or nominate some validators on their behalf.
+// It is not concerned with some important constraints like unbonding period etc.
+
+use gstd::{
+    debug,
+    msg::{self, built_in::staking::*},
+    prelude::*,
+    ActorId,
+};
+use hashbrown::HashMap;
+use hex_literal::hex;
+use parity_scale_codec::Encode;
+
+// Staking proxy built-in actor id (hardcoded for all runtimes)
+const BUILT_IN: ActorId = ActorId::new(hex!(
+    "9d765baea1938d17096421e4f881af7dc4ce5c15bb5022f409fc0d6265d97c3a"
```

Comment:
Why this address is not declared in the `built_in` module?

Reply:
There can potentially be more than one built-in actor who are essentially differentiated by their addresses (actor ids). The idea was that these addresses could be published off chain and kept up-to-date in case new actors are added or the algorithm how we derive an address changes.
But seems that declaring them as `built_in` module constants is a good idea.